{{cookiecutter.project_slug}}/CONTRIBUTORS.txt

```diff
@@ -1 +1 @@
-{{ cookiecutter.author_name }}
+enchance
```

Comment:
Ah that's the wrong `CONTRIBUTORS` file 😄  The one to update is here: https://github.com/pydanny/cookiecutter-django/blob/master/CONTRIBUTORS.rst